ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/tasks/CompleteBlocksTaskTest.java

```diff
@@ -79,6 +95,112 @@ public void shouldCompleteWithoutPeersWhenAllBlocksAreEmpty() {
     assertThat(task.run()).isCompletedWithValue(blocks);
   }
 
+  @Test
+  public void shouldCreateCorrectEmptyBlockWhenWithdrawalsAreEnabled() {
```

Comment:
```suggestion
  public void whenWithdrawalsAreEnabled_shouldCreateEmptyBlockWithEmptyWithdrawalsList() {
```

Reply:
Renamed both tests to be (hopefully) clearer and more consistent